pkg/storage/SQL/operational_solution/and_possible_get_by_operational_need_id_LOADER.sql

```diff
@@ -32,10 +32,10 @@ possibleSolution AS (
         OpSol.modified_dts,
         TRUE AS is_common_solution -- Linked Possible Solutions are always a common solution
     FROM QUERIED_IDS AS qID
-    INNER JOIN operational_need AS OpNd ON OpNd.id = qID.operational_need_id
-    INNER JOIN possible_need_solution_link AS PNSL ON PNSL.need_type = OpNd.need_type
-    INNER JOIN possible_operational_solution AS pOpSol ON pOpSol.id = PNSL.solution_type
-    LEFT JOIN operational_solution AS OpSol ON OpSol.solution_type = pOpSol.id AND OpSol.operational_need_id = OpNd.id
+    INNER JOIN operational_need AS OpNd ON qID.operational_need_id = OpNd.id
+    INNER JOIN possible_need_solution_link AS PNSL ON OpNd.need_type = PNSL.need_type
+    INNER JOIN possible_operational_solution AS pOpSol ON PNSL.solution_type = pOpSol.id
+    LEFT JOIN operational_solution AS OpSol ON pOpSol.id = OpSol.solution_type AND OpNd.id = OpSol.operational_need_id
```

Comment:
Were these changes functional, or just re-ordering?

Reply:
that's correct, this was the SQL fluff linter from VSCode, functionally they are equivalent